docs/hosting-a-server-with-northstar/basic-listen-server.md

```diff
@@ -1,5 +1,8 @@
 # Hosting a Basic Listen Server
 
+A listen server only runs while the user is in the match, and acts like a 'public' private match/lobby\
+The only difference between a listen and dedicated server is that a listen can only run while the host is in the match, while a dedi will run without the user or TF|2 open.
```

Comment:
Going through it pretending to have never hosted a server before, I noticed that "in a match" kinda implies actively have a map loaded instead of sitting in private match lobby being enough.